### ISSUE TITLE
No support for McMillicancuddy's Farm

### ISSUE THREAD

**Issue by mapledyne** [mapledyne/yaaz#4]
Support and "should we do it?" flags should be added

**Comment by Gaikotsu2** [comment]
Completing the farm costs more turns than it saves. If the goal is speedy ascensions, then I would say not to do it.

**Comment by mapledyne** [comment]
True, but it seems fair to want to include so people can flag it on and off - same as the hippy side fight. It's almost always better to fight as a fratboy, but seems reasonable to allow people to flag those quests if they're wanting a specific award. Not high priority but some basic support would be nice.

**Comment by mapledyne** [comment]
(or maybe allow them to pick an award, and that will determine the side quests automatically?)

**Comment by Gaikotsu2** [comment]
Will need to make sure we pick up a chaos butterfly in the castle to reduce the total number of ducks needing to be killed.

**Comment by mapledyne** [comment]
Yeah - I dislike the Farm all around (hence it not being coded at all at this point. :)
